vaccine/hotline_callback.py

```diff
@@ -0,0 +1,272 @@
+import logging
+import re
+from datetime import datetime, timezone
+from typing import List
+from urllib.parse import urljoin
+
+import aiohttp
+import holidays
+import sentry_sdk
+
+from vaccine import hotline_callback_config as config
+from vaccine.base_application import BaseApplication
+from vaccine.models import Message
+from vaccine.states import Choice, EndState, FreeText, WhatsAppButtonState
+from vaccine.utils import (
+    HTTP_EXCEPTIONS,
+    TZ_SAST,
+    display_phonenumber,
+    get_today,
+    normalise_phonenumber,
+)
+from vaccine.validators import nonempty_validator, phone_number_validator
+
+logger = logging.getLogger(__name__)
+za_holidays = holidays.SouthAfrica()
```

Comment:
This is cool

Reply:
Yeah, I started with a list of datetimes for the next few years, but found this library, which handles Easter properly and everything.